pages/support.md

```diff
@@ -4,6 +4,20 @@
 }
 </script>
 
+## Version Support Status
+
+We support only the latest version of jQuery. The 1.x and 2.x branches are no longer supported.
+
+jQuery 3.x is the current version branch of jQuery.
+
+jQuery 4.0 is in beta. Once jQuery 4.0 is released, we will provide security patches and bug fixes on the 3.x branch for a limited time.
+
+#### Still on an old version? Here are your options
```

Comment:
Under H2, is H3 not H4.

For signal-noise reduction and scan-ability, I suggest omitting "Here are your options".